requirements.txt

```diff
@@ -1,2 +1,2 @@
 lightning >=2.0.0
-# bagua >=0.9.2
+bagua-cuda116 >=0.9.2
```

Comment:
@Borda Bagua-cuda116 can also be used in lower versions of the cuda environment, such as cuda 11.5. We have not yet tested it on cuda 11.7. If there are any problems, users can also reinstall the corresponding version of bagua with `pip install bagua-cudaXXX`.

Reply:
> cuda116 can also be used in lower versions of the cuda environment, such as cuda 11.5

Are you sure, because I used to have problems and did not work to run later cuda docker images (which means full cuda was there) on machines with drives from lower cuda

Reply:
@Borda I have tested bagua-cuda116 on CUDA 11.5 and it works. If users encounter errors while using Bagua-cuda116 with lower cuda, they can use 'pip instal bagua-cudaXXX' to install corresponding Bauga version to avoid the errors.